### ISSUE TITLE
Stylus mode: Incorrect upper case hex colors highlighting

### ISSUE THREAD

**Issue by pabli24** [ajaxorg/ace#4958]
### Describe the bug

Hex colors that are written in the upper case have incorrect syntax highlighting

### Expected Behavior

![stylus-mode-hex-colors-expected](https://user-images.githubusercontent.com/22481052/196050174-dfc3d47f-97f1-427e-be5d-b41a4e4b605a.png)

### Current Behavior

![stylus-mode-hex-colors](https://user-images.githubusercontent.com/22481052/196050184-49992a9f-634c-4372-9794-159465a9cdff.png)

### Reproduction Steps

```styl
body
	color: #6A3399
	color: #6a3399
	color: #AAA
	color: #aaa
```

### Ace Version

1.12.0

**Comment by mkslanc** [comment]
[This PR](https://github.com/ajaxorg/ace/pull/4962) should solve the issue

**Comment by andrewnester** [comment]
Fixed and released in 1.12.1
